### PR TITLE
LRSD-4100 [CP 2.0] Update site initializer with new product names

### PR DESCRIPTION
**Pull request by pedroliveirav**
[LRSD-4100](https://liferay.atlassian.net/browse/LRSD-4100)

### CODE REVIEW

**Comment by kylembischof** [comment]
I notice some other old product names in other files, do we need to change the following places as well? Some examples:

notification-templates
account-subscription-group.object-entries.json
account-subscription-term.object-entries.json
account-subscription.object-entries.json

**Comment by pedroliveirav** [comment]
> account-subscription-group.object-entries.json

I had initially understood that we shouldn't change any references in the Objects and that I should stick to changing only the name of the mocked test sites, but I can do that. Will it be necessary to make the changes in UAT as well or only in the initializer site? If it's necessary, it'll have to be changed manually.

**Comment by amosfong** [comment]
> Will it be necessary to make the changes in UAT as well or only in the initializer site? If it's necessary, it'll have to be changed manually.

I think the UAT data would be fixed by our rename scripts. So you can just change site initializer only.

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/notification-templates/template-1/en-US.html [hunk 1]
```diff
@@ -1,6 +1,9 @@
-<h3><strong>Someone submitted the LXC - SM Activation form for [%PROJECT_CODE%] at [%DATE_AND_TIME_SUBMITTED%]</strong></h3>
+<h3>
+<strong>Someone submitted the Liferay PaaS Activation form for [%PROJECT_CODE%]
+		at [%DATE_AND_TIME_SUBMITTED%]</strong>
```
Comment (kylembischof) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/notification-templates/template-1/en-US.html:3]: fix inconsistent indentation

Comment (pedroliveirav) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/notification-templates/template-1/en-US.html]: Done.

**Comment by amosfong** [comment]
ci:forward:force

**Review by amosfong — APPROVED** [review]

**Pull request closed without merging**